dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBEncoderDecoder.java

```diff
@@ -1031,6 +1060,12 @@ private static XMLStreamReader findExtraNamespaces(XMLStreamReader source) {
                 }
             }
             if (!nsMap.isEmpty()) {
+	        if (isLoggableFinest) { // Liberty change begin
+                   LOG.finest("findExtraNamespaces: Got some extra namespaces: " + (nsMap != null ? nsMap.size() : "0"));
+                   for (Map.Entry<String, String> exNs: nsMap.entrySet()) {
+                      LOG.finest("ExNSkey: " + exNs.getKey() + ", ExNSValue: " + exNs.getValue());
```

Comment:
Do we know for sure that `nsMap.toString()` will be unreadable when logged currently? I've concerns about iterating over the entire `nsMap`, seems like it could be excessively large.

Reply:
NsMap is not logged currently, If we log it, it should be readable, I can replace the for loop and just log naMap instead.